src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Suppressions.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is very likely wrong, there will be linker warnings for Mono's SPC codes

Reply:
This is still WIP. I'm aware that there are warnings for Mono's SPC as evidenced by the CI failures. The gaps will be filled in shortly.